creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -0,0 +1,133 @@
+const moment = require('moment')
+
+const redisClient = require('../redis')
+const { logger: genericLogger } = require('../logging')
+const config = require('../config')
+
+const CREATOR_NODE_ENDPOINT = config.get('creatorNodeEndpoint')
```

Comment:
why have this in the prefix? this seems redundant since this redis is meant for this creator node